src/app/view/home/home.component.ts

```diff
@@ -13,4 +13,11 @@ export class HomeComponent implements OnInit {
     ngOnInit() {
     }
 
+    honk() {
+        let audio = new Audio();
+        audio.src = "../../../assets/Honk1.wav";
```

Comment:
Lines 17 + 18 should of been in the constructor. =p

Reply:
HONK, HONK HONK HONK on load? Otherwise HONK HONK HONK on press, correct? HONK.